python/instrumentation/openinference-instrumentation-bedrock/README.md

```diff
@@ -2,10 +2,30 @@
 
 Python autoinstrumentation library for AWS Bedrock calls made using `boto3`.
 
-This package implements OpenInference tracing for `invoke_model` calls made using a `boto3` `bedrock-runtime` client. These traces are fully OpenTelemetry compatible and can be sent to an OpenTelemetry collector for viewing, such as [Arize `phoenix`](https://github.com/Arize-ai/phoenix).
+This package implements OpenInference tracing for `invoke_model` and `converse` calls made using a `boto3` `bedrock-runtime` client. These traces are fully OpenTelemetry compatible and can be sent to an OpenTelemetry collector for viewing, such as [Arize `phoenix`](https://github.com/Arize-ai/phoenix).
```

Comment:
Please add a note to the documentation that shows the minimum boto version required for converse support